src/components/math-symbols/MathSymbol.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+const MATH_SYMBOL_TYPE = {
+  SQUERE_ROOT: 'squere-root',
+  NTH_ROOT: 'nth-root',
+  POWER: 'power',
+  SUBSCRIPT: 'subscript',
+  LESSEQUAL: 'less-then-or-equal',
+  GREATEREQUAL: 'greater-then-or-equal',
+  INEQUALITY: 'inequality',
+  DIVISION: 'division',
+  PI: 'pi',
+  ALPHA: 'alpha',
+  BETA: 'beta',
+  LINE: 'line',
+  LIMIT: 'limit',
+  MATRIX: 'matrix',
+  INTEGRAL: 'integral',
+  EQUATION_SYSTEM: 'equation-system'
+};
+
+const WIDE = [MATH_SYMBOL_TYPE.LIMIT, MATH_SYMBOL_TYPE.MATRIX, MATH_SYMBOL_TYPE.INTEGRAL, MATH_SYMBOL_TYPE.EQUATION_SYSTEM];
+
+const SIZE = {
+  SMALL: 'small',
+  MEDIUM: 'medium',
+  NORMAL: 'normal'
+};
+
+const COLOR = {
+  LIGHT: 'light',
+  ADAPTIVE: 'adaptive',
+  GRAY: 'gray',
+  GRAY_SECONDARY: 'gray-secondary',
+  GRAY_LIGHT: 'gray-light',
+  BLUE: 'blue',
+  MUSTARD: 'mustard',
+  LAVENDER: 'lavender',
+  PEACH: 'peach',
+  DARK: 'dark',
+  MINT: 'mint'
+};
+
+const MathSymbol = ({type = MATH_SYMBOL_TYPE, size = SIZE.NORMAL, color, className, ...props}) => {
+  const iconClass = classNames('sg-math-symbol-icon', {
+    [`sg-math-symbol-icon--${size}`]: size !== SIZE.NORMAL,
+    ['sg-math-symbol-icon--wide']: WIDE.includes(type),
```

Comment:
we can't use `.includes` :( polyfillio doesn't support it - please use `WIDE.indexOf(type) !== -1`